### PR TITLE
Pass PR repo slug to the bender runner

### PR DESCRIPTION
**Pull request by sculpt0r**
## What is the purpose of this pull request?

Bug fix

## Does your PR contain necessary tests?

All patches that change the editor code must include tests. You can always read more
on [PR testing](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_contributing_code.html#tests),
[how to set the testing environment](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html) and
[how to create tests](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html#creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [ ] Manual tests

## Did you follow the CKEditor 4 code style guide?

Your code should follow the guidelines from the [CKEditor 4 code style guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md) which helps keep the entire codebase consistent.

- [ ] PR is consistent with the code style guide

## What is the proposed changelog entry for this pull request?

```
skip
```

## What changes did you make?

Pass PR repo slug to the bender runner as described in: https://github.com/ckeditor/ckeditor4-benderjs-runner/pull/23

### CODE REVIEW

**Comment by jacekbogdanski** [comment]
@sculpt0r how does it integrate with https://github.com/ckeditor/ckeditor4-benderjs-runner/pull/25 ?

Looks like we may have some API issue here, as it depends on the order of the parameters (and we are starting to support a bit too many parameters without using named parameters like `--repo-path`)

**Comment by sculpt0r** [comment]
I added naming parameters to test runner: https://github.com/ckeditor/ckeditor4-benderjs-runner/pull/23

Soon I'll update this PR

**Comment by sculpt0r** [comment]
Rebased on newest master

File: plugins/wysiwygarea/plugin.js [hunk 1]
```diff
@@ -8,177 +8,177 @@
  *		mode, which handles the main editing area space.
```
Comment (jacekbogdanski) [inline plugins/wysiwygarea/plugin.js:8]: What happened here? 😄

Comment (sculpt0r) [reply plugins/wysiwygarea/plugin.js:8]: Oh my... just some f...unky format on save... Sorry about that... This file shouldn't be touched.

Comment (sculpt0r) [reply plugins/wysiwygarea/plugin.js:8]: That is some format on save... however, this file was modified just to prove that bender runner generates a nonempty filter for tests. If everything will be OK - those changes should be removed before merge.

**Review by jacekbogdanski — APPROVED** [review]

**Pull request merged**